pkg/istio/onboarding/controller.go

```diff
@@ -98,7 +98,6 @@ func (c *Controller) processNextItem() bool {
 		}
 	}
 
-	c.queue.Forget(key)
```

Comment:
We no longer have a Forget call anywhere in the onboarding controller now. I believe we need to have similar logic as the main controller where we forget after a sync has a no-op.